astropy/cosmology/utils.py

```diff
@@ -134,8 +134,8 @@ def aszarr(z):
     if isinstance(z, (Number, np.generic)):  # scalars
         return z
     elif hasattr(z, "shape"):  # ducktypes NumPy array
-        if hasattr(z, "unit"):  # Quantity
-            return z.to_value(cu.redshift)  # for speed only use enabled equivs
+        if hasattr(z, "unit"):  # Quantity Column
+            return z.to(cu.redshift).value  # for speed only use enabled equivs
```

Comment:
Huh... I thought these two calls are equivalent, @mhvk ?

Reply:
Yes, the two are equivalent for `Quantity`, but as noted on top, `Column.to_value` does not exist....  The one issue is that if the unit is already right, this will make a copy. To avoid this, one option might be
```
(z << u.redshift).value
```
which should not give too much of a slow-down for `Quantity` and probably is the same for `Column`.

Of course, yet another alternative is simply to ignore the unit on a `Column`, as is done in most other cases (much to my chagrin!).

Reply:
Thanks for the suggestion. I just did a quick test and it looks like `(z << u.redshift).value` is faster for both `Quantity` and `Column`.